lib/features/thread/presentation/widgets/scroll_to_top_button_widget.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:core/presentation/utils/responsive_utils.dart';
+import 'package:flutter/material.dart';
+import 'package:tmail_ui_user/features/thread/presentation/styles/scroll_to_top_button_widget_styles.dart';
+
+class ScrollToTopButtonWidget extends StatefulWidget {
+  final ScrollController scrollController;
+  final GestureTapCallback onTap;
+  final ResponsiveUtils responsiveUtils;
+  final double? limitIndicator;
+  final double? elevation;
+  final Color? colorButton;
+  final double? buttonRadius;
```

Comment:
If you have set a `default initialization value` for it, there is no need for the `nullable` data type